flake.nix

```diff
@@ -33,17 +38,32 @@
           packages.default = pkgsOverlayed.paretosecurity;
 
           checks = {
+            autologin = pkgsOverlayed.testers.runNixOSTest ./test/integration/autologin.nix;
             cli = pkgsOverlayed.testers.runNixOSTest ./test/integration/cli.nix;
             firewall = pkgsOverlayed.testers.runNixOSTest ./test/integration/firewall.nix;
             help = pkgsOverlayed.testers.runNixOSTest ./test/integration/help.nix;
             luks = pkgsOverlayed.testers.runNixOSTest ./test/integration/luks.nix;
-            zfs = pkgsOverlayed.testers.runNixOSTest ./test/integration/zfs.nix;
             pwd-manager = pkgsOverlayed.testers.runNixOSTest ./test/integration/pwd-manager.nix;
-            screenlock = pkgsOverlayed.testers.runNixOSTest ./test/integration/screenlock.nix;
             secureboot = pkgsOverlayed.testers.runNixOSTest ./test/integration/secureboot.nix;
             trayicon = pkgsOverlayed.testers.runNixOSTest ./test/integration/trayicon.nix;
             xfce = pkgsOverlayed.testers.runNixOSTest ./test/integration/desktop/xfce.nix;
-            autologin = pkgsOverlayed.testers.runNixOSTest ./test/integration/autologin.nix;
+            zfs = pkgsOverlayed.testers.runNixOSTest ./test/integration/zfs.nix;
+
+            screenlock = pkgsOverlayed.testers.runNixOSTest ./test/integration/screenlock.nix;
+            screenlock-plasma5 =
```

Comment:
This sadly needs to be a separate file to get the old 24.11 nixpkgs in there.

If someone can figure out how to merge this into `screenlock.nix`, I'm all ears!